README.md

```diff
@@ -18,6 +18,12 @@ Article templates used within the Guardian’s next-generation iOS, Android and
 * run `yarn` to install dependencies.
 * run `yarn setup` to locally ignore build files so they are not checked into master
 
+## IOS and Android devs
+If you are developing against a branch which is not `release`, please follow the steps below:
+* Pull down the branch which you are developing against
+* Please follow the requirements and developing steps
+* run `yarn sync` which will build the project, and move the build into the individual Android and IOS projects
```

Comment:
They'll need to run 'yarn build' to generate the assets before copying them with 'yarn sync'